### PR TITLE
doc: nautilus 14.2.2 release notes, take three

### PR DESCRIPTION
**Pull request by smithfarm**
This reverts the changes made by https://github.com/ceph/ceph/pull/29145 and adds the PRs that were missing from https://github.com/ceph/ceph/pull/29011.

### CODE REVIEW

**Comment by theanalyst** [comment]
jenkins render docs

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -1446,7 +1351,7 @@ Detailed Changelog
 * ceph-volume: do not pin the testinfra version for the simple tests (`pr#23268 <https://github.com/ceph/ceph/pull/23268>`_, Andrew Schoen)
 * ceph-volume: do not send (lvm) stderr/stdout to the terminal, use the logfile (`issue#36492 <http://tracker.ceph.com/issues/36492>`_, `pr#24738 <https://github.com/ceph/ceph/pull/24738>`_, Alfredo Deza)
 * ceph-volume do not use stdin in luminous (`issue#25173 <http://tracker.ceph.com/issues/25173>`_, `pr#23355 <https://github.com/ceph/ceph/pull/23355>`_, Alfredo Deza)
-* ceph-volume: don't create osd['block.db'] by default (`issue#38472 <http://tracker.ceph.com/issues/38472>`_, `pr#26627 <https://github.com/ceph/ceph/pull/26627>`_, Jan Fajerski)
+* ceph-volume: don't create osd['block.d] by default (`issue#38472 <http://tracker.ceph.com/issues/38472>`_, `pr#26627 <https://github.com/ceph/ceph/pull/26627>`_, Jan Fajerski)
```
Comment (theanalyst) [inline doc/releases/nautilus.rst]: accidental change?

File: doc/releases/nautilus.rst [hunk 2]
```diff
@@ -997,7 +902,7 @@ These changes occurred between the Mimic and Nautilus releases.
 * ``rados df [detail]`` output (POOLS section) has been modified in json
   format:
   
-  - 'size_bytes' and 'size_kb' columns now show the space (accumulated
+  - 'size_bytes' and 'size_k columns now show the space (accumulated
```
Comment (theanalyst) [inline doc/releases/nautilus.rst]: accidental change?

**Comment by theanalyst** [comment]
jenkins build docs

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
jenkins test docs

**Comment by smithfarm** [comment]
jenkins test docs

**Pull request merged**